examples/stable_diffusion_v2/tools/safety_checker/safety_checker.py

```diff
@@ -0,0 +1,275 @@
+"""
+Implementation of two versions of safety checker in stable
+diffusion 1/2, respectively
+"""
+import argparse
+import os
+import sys
+
+import numpy as np
+import yaml
+from PIL import Image
+
+# equivalent to no-check-certificate flag in wget
+os.environ["CURL_CA_BUNDLE"] = ""
```

Comment:
It is not safe to disable certificate verification for everyone. Perhaps you can describe it as an option in `safety_check.md`?

Reply:
fixed